Description: Pass --no-readline to R otherwise Pexpect hangs
Author: Ximin Luo <infinity0@debian.org>
Forwarded: TODO, maybe not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sage/src/sage/interfaces/r.py
+++ b/sage/src/sage/interfaces/r.py
@@ -331,7 +331,7 @@
                   prompt = '> ', #default, later comes the change
 
                   # This is the command that starts up your program
-                  command = "R --vanilla --quiet",
+                  command = "R --no-readline --vanilla --quiet",
 
                   server=server,
                   server_tmpdir=server_tmpdir,
